style.css

```diff
@@ -184,22 +207,32 @@ footer {
     justify-content: center;
     align-items: center;
     height: 600px;
-    width: 1000px; 
+    width: 1000px;
     padding: 40px;
     background-color: #FE9D8D;
     border: 5px #373C99 solid;
     z-index: 10;
+    opacity: 0.80;
```

Comment:
the problem with using opacity is that it also applies to the text and the button.

instead, you should use hexa or rgba for the background colour, which allows you to have a transparent background with fully opaque text and button.

so if you change your background colour from `#FE9D8D` to `rgba(254, 157, 141, 0.5)` and remove the `opacity`, it will look exactly the way i want